integrations/dynatrace/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- towncrier release notes start -->
```

Comment:
The changelog content for this feature is missing. You can use this steps to generate a changelog

towncrier create --content "Implemented Dynatrace integration to bring slos, entities and problems data to Port" 0.1.0.feature.md

commit the change log files
towncrier build --yes --version 0.1.16


repeat for each new feature/PR